src/platform/packages/shared/kbn-scout/src/playwright/fixtures/worker/scout_space/parallel.ts

```diff
@@ -130,11 +126,12 @@ export const scoutSpaceParallelFixture = coreWorkerFixtures.extend<
         setDefaultTime,
       };
 
-      log.serviceLoaded(`scoutSpace:${spaceId}`);
+      log.serviceMessage('scoutSpace', `New Kibana space '${spaceId}' created`);
```

Comment:
I searched the `scoutSpace` string and it's only appearing once - with that said, it feels to me that in the future we may want to abstract it away in some place so we don't repeat it. 

Perhaps in the `@kbn/scout-info` package, but I'm not yet too sure. What do you think?

Reply:
Ah, understood! Thank for clarifying.